clcache.py

```diff
@@ -1371,24 +1379,24 @@ def processCacheHit(cache, objectFile, cachekey):
         return 0, cachedArtifacts.stdout, cachedArtifacts.stderr, False
 
 
-def createManifest(manifestHash, includePaths):
-    baseDir = normalizeBaseDir(os.environ.get('CLCACHE_BASEDIR'))
```

Comment:
Any particular reason you moved this code into `collapseBasedirToPlaceholder` (and `expandBasedirPlaceholder`)? That seems to repeat the `normalizeBaseDir(os.environ.get(..))` call for every included files, doesn't it? I.e. the older version seems more efficient to me on first glance.